src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml

```diff
@@ -1,6 +1,6 @@
 # SSD MobileNetV2 Detection training on CoCo2017 Dataset:
 # Trained in 320x320 mAP@0.5@0.95 (COCO API, confidence 0.001, IoU threshold 0.6, test on 320x320 images) ~20.52
-# Checkpoint path: https://deci-pretrained-models.s3.amazonaws.com/ssd_lite_mobilenet_v2_coco_res320_new_coco_filtered_affine_scale_5_15_no_mosaic/ckpt_best.pth
+# Checkpoint path: https://deci-pretrained-models.s3.amazonaws.com/ssd_lite_mobilenet_v2/coco2017/2022-11-28/average_model.pth
 # (trained with stride_16_plus_big)
 # Hardware: 8 NVIDIA RTX 3090
```

Comment:
Didn't we train with 4 devices ? Can you update the comments in the YAML file accordingly ?

Reply:
Updated, thanks